zuul.d/project.yaml

```diff
@@ -44,6 +44,18 @@
             files:
               - playbooks/ansible-network-appliance-base/.*
               - playbooks/ansible-network-vyos-appliance/.*
+        - molecule-tox-py27-centos-7: &tox
+            files:
+              - ^bindep.txt$
+              - ^requirements.txt$
+              - ^test-requirements.txt$
+              - ^tox.ini$
+              - playbooks/ansible-tox-molecule/.*
+            vars:
+              tox_extra_args: --notest
+        - molecule-tox-py36-centos-8: *tox
+        - molecule-tox-py36-ubuntu-bionic: *tox
+        - molecule-tox-py37-fedora-30: *tox
```

Comment:
I am unsure we want to add these jobs here. I wonder if we should maybe consider shared zuul jobs for molecule? Which then you and molecule test directly control

Reply:
In fact these jobs do not have any "molecule" on them but they add "docker" and "podman" which is needed by molecule jobs.

These jobs are already used in other projects, but they are defined in molecule one. I have nothing against moving them under ansible-zuul-jobs if you want, as I see them as multi-purpose.

Still there is one thing that I will continue to want: to run them when we update the pre.yaml file, so we can easily confirm that the patch is correct. This is why I had the special pattern added, as I did not want to trigger them on random changes, only on those that could break them.